examples/cra/src/components/IndicatorList.js

```diff
@@ -16,6 +16,7 @@ const query = {
 
 export const IndicatorList = () => {
     const { loading, error, data, refetch } = useDataQuery(query)
+    const show = useAlert(id => `Created indicator ${id}`)
```

Comment:
`useAlert` only returns the `show` function. I don't think anything else would be needed by the app and the big advantage is that the dev can now choose a variable name. This can be very handy when one component can show multiple alerts, i.e.:
```javascript
const showGoodAlert = useAlert('Yippeee!')
const showBadAlert = useAlert('Boooh')
```

Reply:
It's a little less ergonomic but it is possible to choose a variable name with an object as well, or to return an array (a la `setState`) :

```js
const { show: showGoodAlert } = useAlert('Yippeee!', { success: true })
const { show: showBadAlert } = useAlert('Boooh', { critical: true })

// Or with an array
const [showGoodAlert] = useAlert('Yippeee!', { success: true })
const [showBadAlert] = useAlert('Boooh', { critical: true })
```

I worry about returning just a function because it requires a breaking change to extend the API in the future.